src/string_bitflip.cc

```diff
@@ -0,0 +1,20 @@
+#include "string_bitflip.h"
+
+#include "node.h"
+#include "v8.h"
+
+#include <string.h>
+#include <limits.h>
+
+namespace node {
+
+void StringBitflip::BE_FlipBits (uint16_t* dst,
+                                 const uint16_t* src,
+                                 size_t buflen) {
+
+  for (size_t i = 0; i < buflen; i++) {
+    dst[i] = (src[i] << 8) | (src[i] >> 8);
+  }
+
+};
```

Comment:
Oh, another thing: I'd put this in util.h and util-inl.h.  Having it in a separate compilation unit means the compiler won't know if src and dst alias, it will always have to assume the pessimistic case.  (Maybe not true with LTO but we don't build with LTO currently.)